lib/measures/tbd/tests/tbd_tests.rb

```diff
@@ -180,7 +185,9 @@ def test_load_tbd_json_error
     # assert that it ran correctly
     assert_equal('Fail', result.value.valueName)
     assert(result.errors.size == 1)
-    assert(result.warnings.empty?)
+    assert(result.warnings.size == 1)
+    log_message = "Can't find 'tbd.json' - simulation halted"
+    assert(result.warnings[0].logMessage == log_message)
```

Comment:
@macumber : Please suggest additional tests (if any) we should be implementing ...

Reply:
I think you have good coverage in the gem spec, these tests are really just to make sure the measure isn't broken, I think what you have here is fine